src/panels/lovelace/cards/hui-tile-card.ts

```diff
@@ -331,8 +331,6 @@ export class HuiTileCard extends LitElement implements LovelaceCard {
             class="background"
             @action=${this._handleAction}
             .actionHandler=${actionHandler()}
-            role="button"
-            tabindex="0"
```

Comment:
@piitaya please double check this.  I'm removing the button role and tabindex here because the `div` is empty and the following `div` is the button as it should be I think, so there should only be one tab stop.

If I'm reading the CSS right, this is just the background stacked underneath the actual content in the same space, so it has no reason to be focused?  Why is the content not just inside this background `div`?

Reply:
The role button is here because there are 2 buttons in this card : 
- the icon that toggle the entity
- the full card that open the more info

It's the same behavior than Apple Home tiles. 

I created a absolute background behind to avoid button inside button because if it's not valid for accessibility. Do you know how we can have this behavior with a valid accessibility support?.

Reply:
Ah... okay, then yeah they shouldn't be nested.  The `@action` is handling the click/spacebar I presume?

I think just to fix this, let's add:
1. If the entity name is visible, `aria-labeledby` should point to it, otherwise
2. `aria-label=${<entity-name>}`

I'll revert my commit.  Can you add the label or suggest the change?

This is kind of a systemic issue throughout the UI.  Clicking all types of places opens the more info dialogs, but most of those places don't have roles or tab stops.  At least this one did. 👍🏻